aiida_firecrest/scheduler.py

```diff
@@ -276,12 +290,15 @@ def get_jobs(
                     )
                 )
 
+            # TODO: The block below is commented, because the number of
+            # allocated cores is not returned by the FirecREST server
```

Comment:
I don't fully understand? Can we maybe explain this in an issue more explicitly?

Reply:
Sure:
```suggestion
            # `aiida-ssh` transport plugin parses two additional key values: `num_mpiprocs` & `num_machines`
            # from Slurm's `squeue` response. The later is number of nodes in use, and the former is number of cpus.     
            # `FirecREST` server however doesn't return the raw response of `squeue` to use, but parse it 
            # themselves and return a modified raw response, in which they don't specify `num_mpiprocs`.
```

In anycase, I'm not aware of any crucial usecase in `aiida-core` for `num_mpiprocs`. 
 Therefore we just document here for future if needed.

Reply:
Yes, FirecREST doesn't return this info. 
Ok I opened an issue on this, and removed from here
https://github.com/aiidateam/aiida-firecrest/issues/39